doc/source/timeseries.rst

```diff
@@ -265,17 +265,34 @@ Typical epoch stored units
    pd.to_datetime([1349720105100, 1349720105200, 1349720105300,
                    1349720105400, 1349720105500 ], unit='ms')
 
-These *work*, but the results may be unexpected.
+.. note::
 
-.. ipython:: python
+   Epoch times will be rounded to the nearest nanosecond.
+
+.. warning::
+
+   Conversion of float epoch times can lead to inaccurate and unexpected results.
+   Python floats are unbounded precision, with about 15 significant digits (depends on
```

Comment:
What do you mean with "unbounded precision"? that seems to contradict a bit with "15 digits"